payment-intent/cs/payment-intent-manual.cs

```diff
@@ -0,0 +1,17 @@
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
+var result = await paymentIntentsClient.CreatePaymentIntentAsync(new CreatePaymentIntentRequest
+{
+    Amount = new()
+    {
+        Value = checkoutRequest.Amount,
+        CurrencyCode = "GBP"
+    },
+    Config = new()
+    {
+        CancelUrl = new Uri(checkoutRequest.CancelUrl),
+        RedirectUrl = new Uri(checkoutRequest.RedirectUrl),
+    },
+    Description = checkoutRequest.Description,
+    Reference = "Order - 1",
+    captureMode = captureMode.Manual
```

Comment:
CaptureMode.Manual it is case sensitive

Reply:
we need at least to manually to run samples before puting them here
if you can't yourself - please involve devs